roles/coreos-bootstrap/files/runner

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
-BINDIR="/usr/local/bin"
+BINDIR="/opt/bin"
```

Comment:
don't move the file from preinstall, do a different PR to cleanup if not required by other OS